guide/content/stylesheets/components/_example.sass

```diff
@@ -32,5 +32,5 @@
 .final-decision-warning
   @extend %govuk-body-s
 
-  background: govuk-colour("pink")
-  padding: govuk-spacing(2)
\ No newline at end of file
+  background: govuk-tint(govuk-colour('pink'), 92%)
+  padding: govuk-spacing(2)
```

Comment:
Ah yes. I was having trouble reloading my styles locally. Was thinking of having more obvious styling for these classes, using background colours and contrasting text. So this would have a pink background and white text.

Reply:
Ah I noticed the styles not refreshing too, will take a look now.

Reply:
I've given you access to this repo, please feel free to commit or raise PRs directly

Reply:
Tried white on the GOV.UK shade of pink but that didn't have necessary contrast either. Live reload should be working again now.

I'm going to make the Components version of the guide reflect all these changes and then  when the dust has settled look at standardising things (maybe the styles into a npm package) to try and reduce any duplication.